Donation-platform/src/main/webapp/frontend/src/test.js

```diff
@@ -0,0 +1,13 @@
+var googleTrends = require('/home/anerishah/step-donation/step28-2020/Donation-platform/src/main/webapp/frontend/node_modules/google-trends-api/lib/google-trends-api.min.js');
+
+/* ******************* Autocomplete **************************/
```

Comment:
This seems to be a test file. I'm not sure if it's a TODO or something, but are there planned to be tests here?

Reply:
that was something that I was using to verify that the Google Trends API was working. I've deleted this file now.